<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>500</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Access Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/icons/appicon</normaloff>:/icons/appicon</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="loadDeviceListAction"/>
    <addaction name="saveDeviceListAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Оборудование</string>
    </property>
    <widget class="QMenu" name="menuBatchUpdate">
     <property name="title">
      <string>Пакетное обновление</string>
     </property>
     <addaction name="updateAllDslamBoardsInfoAction"/>
     <addaction name="updateVlanAllSwitchAction"/>
     <addaction name="updateAllProfileOltInfoAction"/>
     <addaction name="separator"/>
     <addaction name="updateInfoAllDevicesAction"/>
    </widget>
    <addaction name="currentTabAction"/>
    <addaction name="separator"/>
    <addaction name="updateServiceDataAction"/>
    <addaction name="separator"/>
    <addaction name="menuBatchUpdate"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Справка</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Конфигурация</string>
    </property>
    <addaction name="settingsAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menu_2"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aboutAction">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/configure.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Настройки...</string>
   </property>
  </action>
  <action name="loadDeviceListAction">
   <property name="text">
    <string>Загрузить список устройств</string>
   </property>
  </action>
  <action name="saveDeviceListAction">
   <property name="text">
    <string>Сохранить список устройств</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="updateAllDslamBoardsInfoAction">
   <property name="text">
    <string>Досок на дсламах...</string>
   </property>
   <property name="toolTip">
    <string>Обновить информацию о досках на всех dslam'ах...</string>
   </property>
  </action>
  <action name="updateVlanAllSwitchAction">
   <property name="text">
    <string>Вланов на коммутаторах...</string>
   </property>
   <property name="toolTip">
    <string>Обновить информацию о вланах на всех коммутаторах...</string>
   </property>
  </action>
  <action name="updateInfoAllDevicesAction">
   <property name="text">
    <string>Информации на всех устройствах...</string>
   </property>
  </action>
  <action name="updateAllProfileOltInfoAction">
   <property name="text">
    <string>Профилей на олт...</string>
   </property>
  </action>
  <action name="updateServiceDataAction">
   <property name="text">
    <string>Обновить информацию об устройстве</string>
   </property>
  </action>
  <action name="currentTabAction">
   <property name="text">
    <string>Текущая вкладка</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
